pkg/opsgenieclient/client_alert_test.go

```diff
@@ -0,0 +1,116 @@
+package opsgenieclient
+
+import (
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/giantswarm/micrologger/microloggertest"
+)
+
+func Test_GetUnixTime(t *testing.T) {
+	testCases := []struct {
+		name           string
+		input_time     time.Time
+		input_shift    int
+		expectedOutput int64
+	}{
+		{
+			name:           "case 0: normal date, no shift",
+			input_time:     time.Date(2019, 2, 10, 9, 0, 0, 0, time.UTC),
+			input_shift:    0,
+			expectedOutput: 1549789200000,
+		},
+		{
+			name:           "case 0: normal date, 1 day shift",
+			input_time:     time.Date(2019, 2, 10, 9, 0, 0, 0, time.UTC),
+			input_shift:    1,
+			expectedOutput: 1549702800000,
```

Comment:
Seeing this I think the implementation would have been better when the result would be a time object. Then you can do whatever you want with it. Like, getting the unix seconds. I am not sure if this magic number is now correct or not.  No show stopper, but maybe food for thought for the next time.

Reply:
The irritating this here is that the opsgenie queries require unix time _in milliseconds_, which just leads to gross unit conversions everywhere. Agree this function is way too magic, but no 'better' thoughts come to mind :D